test/wv/html_fixtures/oval.html

```diff
@@ -1,15 +1,7 @@
 <div id="2" style="display:flex;flex-direction:row;flex-wrap:wrap;align-content:flex-start;justify-content:flex-start;align-items:flex-start;width:100%;height:100%">
   <div id="3" style="display:flex;flex-direction:row;flex-wrap:wrap;align-content:flex-start;justify-content:flex-start;align-items:flex-start;width:100%">
     <p id="4" style="font-size:12px">Positional arguments:</p>
-    <div id="5" style="position:absolute;top:30px;left:30px;width:160px;height:200px"><svg width="160" height="200" style="fill:black;"><ellipse cx="80" cy="100" rx="80" ry="100" style="stroke:black;stroke-width:2;" /></svg></div>
-  </div>
-  <div id="6" style="display:flex;flex-direction:row;flex-wrap:wrap;align-content:flex-start;justify-content:flex-start;align-items:flex-start;width:100%">
-    <p id="7" style="font-size:12px">As a circle</p>
-    <div id="8" style="position:absolute;top:30px;left:30px;width:160px;height:160px"><svg width="160" height="160" style="fill:pink;"><ellipse cx="80" cy="80" rx="80" ry="80" style="stroke:blue;stroke-width:2;" /></svg></div>
-  </div>
-  <div id="9" style="display:flex;flex-direction:row;flex-wrap:wrap;align-content:flex-start;justify-content:flex-start;align-items:flex-start;width:100%">
-    <p id="10" style="font-size:12px">Keyword arguments:</p>
-    <div id="11" style="position:absolute;top:20px;left:20px;width:90px;height:160px"><svg width="90" height="160" style="fill:green;"><ellipse cx="45" cy="80" rx="45" ry="80" style="stroke:red;stroke-width:4;" /></svg></div>
+    <div id="5" style="position:absolute;top:30px;left:30px;width:160px;height:200px"><svg width="160" height="200" style="fill:yellow;"><ellipse cx="80" cy="100" rx="80" ry="100" style="stroke:rgba(0, 128, 0, 255);stroke-width:2;" /></svg></div>
```

Comment:
@noahgibbs - this fixture (with changed example) shows what I'm trying to achieve to fix oval colors. and it might be a wider pattern. I've implemented the "stringified" method above as a starter example, I know it's similar to our other method. But for HTML we need this to render as a string "rgba(1,2,3,255)" for a 4 length array and "rgb" for a three length. This is what we get if a user does the raw method `green` or `red`

but if they write the string "red" or "green" fine, pass it along. 

I appreciate I might be adding some things to colors that are HTML specific. But for now I'm happy to keep it to a method.

And I guess for shoes_styles I add the formatting like I did above, and then I just repeat the pattern again for "dc" (draw context)?

I'm okay for things to get really repeatable for now if they have to. I actuall would want to feel the pain for a while before abstracting as opposed to premature aggressive abstraction.

But yeah, Just monkeying around with this to get the method colors to work for ovals and seem to have gotten it to behave in this example. Would love your feedback before I finish this work though.

Reply:
We do something similar with rgb_to_hex in calzini.rb, but I like the idea of doing the colour mapping more in Lacci like this. Maybe convert it to an array of 3-4 integers, or otherwise make it more uniform? Right now we do a lot of handling weird cases in Calzini -- you can see me fixing some of that in my PR, but we have a lot of little weird bugs. Having fewer colour formats come from Lacci will make it easier...

I'd like to get rid of draw_context soon -- not completely, but it would become something in Lacci, not something the display service would ever see. Having to check it separately everywhere is annoying. But instead we could just set anything with a recognisable property from the draw context (fill, stroke, strokewidth, etc) to the right value in Drawable#initialize when creating the drawable, and draw context would go away as a style. We couldn't do that until we had positional args for the constructor, but now we do.

Reply:
Nothing wrong with playing around, though. Similarly, no reason we need to rush to get my PR checked in -- there are multiple approaches here. And again, I like yours better for colour properties.